paper/paper.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Game Theory and Python: An educational tutorial to game theory and
+repeated games using Python"
+tags:
+- game theory
+- repeated games
+- prisoner's dilemma
+- axelrod python
+authors:
+- name: Nikoleta E. Glynatsi
```

Comment:
This doesn't look correct. Take a look at https://raw.githubusercontent.com/drvinceknight/Nashpy/master/paper.md